client/src/pages/wizard/Onboard8.tsx

```diff
@@ -1,32 +1,81 @@
-import React, { ReactElement } from 'react';
-import { Button } from 'semantic-ui-react';
+import React, { ReactElement, Dispatch, SetStateAction, FC } from 'react';
+import { Button, Modal, ModalContent } from 'semantic-ui-react';
 import { openPopupWidget } from 'react-calendly';
 
 import '../../css/wizard/Onboard8.css';
 import Onboard8SVG from '../../assets/onboard8.svg';
 import RequiredSVG from '../../assets/required.svg';
+import CloseSVG from '../../assets/close.svg';
+
+interface IProps {
+  isModalOpen: boolean;
+  setModalOpen: Dispatch<SetStateAction<boolean>>;
+  setScheduleConfirmed: Dispatch<SetStateAction<boolean>>;
+  handlePageNext(): void;
+}
 
 /**
  * Builds and controls the form management for Onboard8 of the Onboarding Wizard
  */
-const Onboard8 = (): ReactElement => (
-  <div className="onboard8-wrapper">
-    <img className="page-svg" alt="Oboard 8" src={Onboard8SVG} />
-    <div className="onboard8-content">
-      <div className="page-text">
-        Please schedule an Onboarding Session with a Staff Member.
-        <img className="required-icon" alt="required" src={RequiredSVG} />
-      </div>
-      <Button
-        className="calendly-btn"
-        onClick={() =>
-          openPopupWidget({ url: 'https://calendly.com/sawhney4/60min' })
-        }
+const Onboard8: FC<IProps> = ({
+  isModalOpen,
+  setModalOpen,
+  setScheduleConfirmed,
+  handlePageNext,
+}): ReactElement => {
+  const handleScheduleConfirmation = (): void => {
+    setScheduleConfirmed(true);
+    setModalOpen(false);
+    handlePageNext();
+  };
+
+  return (
+    <div className="onboard8-wrapper">
+      <Modal
+        onClose={() => setModalOpen(false)}
+        className="confirmation-modal"
+        open={isModalOpen}
       >
-        Schedule meeting
-      </Button>
+        <ModalContent className="confirmation-modal-content">
+          <Button
+            onClick={() => setModalOpen(false)}
+            className="close-icon"
+            icon
+          >
+            <img src={CloseSVG} alt="X" />
```

Comment:
I suggest you use Semantic UI React's modal with the default close icon (https://react.semantic-ui.com/modules/modal/#content-close-icon) I just asked Mustafa to move the close icons to the top right cuz it's more conventional too